redigomock.go

```diff
@@ -263,6 +266,24 @@ func (c *Conn) Receive() (reply interface{}, err error) {
 func (c Conn) Stats(cmd *Cmd) int {
 	c.statsMut.RLock()
 	defer c.statsMut.RUnlock()
-	
+
 	return c.stats[cmd.hash()]
 }
+
+// ExpectationsWereMet can guarantee that all commands that was set on unit tests
+// called or call of unregistered command can be caught here too
+func (c Conn) ExpectationsWereMet() error {
+	errMsg := ""
+
+	for _, cmd := range c.commands {
+		if !cmd.Called {
+			errMsg = fmt.Sprintf("%sCommand %s with arguments %#v expected but never called.\n", errMsg, cmd.Name, cmd.Args)
+		}
+	}
+
+	if errMsg != "" {
+		return fmt.Errorf("%s", errMsg)
+	} else {
```

Comment:
You could also remove this `else` and just return nil. Some Go lint tools will give warnings about this.

```go
if errMsg != "" {
  return fmt.Errorf("%s", errMsg)
}

return nil
```